payments-core/src/main/java/com/stripe/android/payments/paymentlauncher/PaymentLauncher.kt

```diff
@@ -46,47 +47,52 @@ interface PaymentLauncher {
         /**
          * Create a [PaymentLauncher] instance with [ComponentActivity].
          *
-         * This API registers an [ActivityResultLauncher] into the [ComponentActivity],  it needs
-         * to be called before the [ComponentActivity] is created.
+         * This API registers an [ActivityResultLauncher] into the [ComponentActivity], it needs
+         * to be called before the [ComponentActivity] is created. Use this API to access specific
+         * beta features.
          */
         fun create(
             activity: ComponentActivity,
             publishableKey: String,
             stripeAccountId: String? = null,
-            callback: PaymentResultCallback
-        ) = PaymentLauncherFactory(activity, callback).create(publishableKey, stripeAccountId)
+            betas: Set<StripeApiBeta> = setOf(),
```

Comment:
I am not sure of a good need to expose the beta headers to our users, and so would lean toward not changing our public api.